spec/clowne/adapters/active_record/associations/belongs_to_spec.rb

```diff
@@ -0,0 +1,146 @@
+describe Clowne::Adapters::ActiveRecord::Associations::BelongsTo, :cleanup, adapter: :active_record do
```

Comment:
To be honest, `associations/has_one_spec.rb` not the best example for associations specs. Mainly because of custom cloners (e.g. `AR::TopicCloner`) which make specs closest to integration tests instead of unit tests. Could you use `associations/has_and_belongs_to_many_spec.rb` as a reference for `associations/belongs_to_spec.rb` (I mean contexts blocks)? It will make this file shorter and clearer.